pkg/inject/server.go

```diff
@@ -110,62 +122,67 @@ func HandleAdmissionRequest(req *v1beta1.AdmissionRequest) v1beta1.AdmissionResp
 
 		ServiceAccountTokenVolumeName, err := getServiceAccountTokenVolumeName(&pod)
 		if err != nil {
 			return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s", pod.Name, req.Namespace, err.Error()))
 		}
 
 		sidecarConfig = generateSecretlessSidecarConfig(
-			secretlessConfig,
-			conjurConnConfigMapName,
-			conjurAuthConfigMapName,
-			ServiceAccountTokenVolumeName)
+			SecretlessSidecarConfig{
+				secretlessConfig:              secretlessConfig,
+				conjurConnConfigMapName:       conjurConnConfigMapName,
+				conjurAuthConfigMapName:       conjurAuthConfigMapName,
+				serviceAccountTokenVolumeName: ServiceAccountTokenVolumeName,
+				sidecarImage:                  sidecarInjectorConfig.SecretlessContainerImage,
+			},
+		)
 		break
 	case "authenticator":
-        conjurAuthConfigMapName, err := getAnnotation(&pod.ObjectMeta, annotationConjurAuthConfigKey)
-        if err != nil {
-            return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s", pod.Name, req.Namespace, err.Error()))
-        }
-
-        conjurConnConfigMapName, err := getAnnotation(&pod.ObjectMeta, annotationConjurConnConfigKey)
-        if err != nil {
-            return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s", pod.Name, req.Namespace, err.Error()))
-        }
-
-        switch containerMode {
-        case "sidecar", "init", "":
-            break;
-        default:
-            return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s value (%s) not supported", pod.Name, req.Namespace, annotationContainerModeKey, containerMode))
-        }
-
-        sidecarConfig = generateAuthenticatorSidecarConfig(AuthenticatorSidecarConfig{
-            conjurConnConfigMapName: conjurConnConfigMapName,
-            conjurAuthConfigMapName: conjurAuthConfigMapName,
-            containerMode:           containerMode,
-            containerName:           containerName,
-        })
-
-        containerVolumeMounts := ContainerVolumeMounts{}
-        for _, receiveContainerName := range conjurTokenReceivers {
+		conjurAuthConfigMapName, err := getAnnotation(&pod.ObjectMeta, annotationConjurAuthConfigKey)
+		if err != nil {
+			return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s", pod.Name, req.Namespace, err.Error()))
+		}
+
+		conjurConnConfigMapName, err := getAnnotation(&pod.ObjectMeta, annotationConjurConnConfigKey)
+		if err != nil {
+			return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s", pod.Name, req.Namespace, err.Error()))
+		}
+
+		switch containerMode {
+		case "sidecar", "init", "":
+			break;
+		default:
+			return failWithResponse(fmt.Sprintf("Mutation failed for pod %s, in namespace %s, due to %s value (%s) not supported", pod.Name, req.Namespace, annotationContainerModeKey, containerMode))
```

Comment:
Line too long. Same with other 2 errors above it.

Reply:
I fixed this in multiple places but it's really out of scope. This project isn't linted nor well-formatted. It'll have to get that love in another PR.

Reply:
Fair enough for old code but these 3 errors were new code so with your new changes to these there's nothing else needed for formatting afaict.